quickjs.c

```diff
@@ -1918,9 +1893,14 @@ void JS_SetMemoryLimit(JSRuntime *rt, size_t limit)
     rt->malloc_state.malloc_limit = limit;
 }
 
-void JS_SetDumpFlags(JSRuntime *rt, uint64_t flags)
+int JS_SetDumpFlags(JSRuntime *rt, uint64_t flags)
 {
+#ifdef ENABLE_DUMPS
+    return 0;
     rt->dump_flags = flags;
+#else
+    return -1;
+#endif
```

Comment:
This seems broken: the function returns before setting the flags

Reply:
Oops. Thanks for catching that